llama_index/node_parser/text/sentence.py

```diff
@@ -181,11 +182,8 @@ def _split_text(self, text: str, chunk_size: int) -> List[str]:
             CBEventType.CHUNKING, payload={EventPayload.CHUNKS: [text]}
         ) as event:
             splits = self._split(text, chunk_size)
-            chunks = self._merge(splits, chunk_size)
```

Comment:
Can we keep the callback and `_merge` in here? Or was there a reason for moving it?

Reply:
ahhhh my bad! we should keep that. I removed the `event.on_end` bit and lint complained and said I don't need to assign variable `chunks`.

reverting this back to way it was!